buildsystem/CompileSuite/autoTests/new_commits.sh

```diff
@@ -93,7 +93,7 @@ touch "$thisDir"runGuard
             # modify compile environment (forwarded to CMake)
             #export PIC_COMPILE_SUITE_CMAKE="-DPIC_ENABLE_PNG=OFF -DCUDA_ARCH=sm_35"
             . /etc/profile
-            module load gcc/4.6.4 boost/1.55.0 cmake/2.8.12.2 cuda/6.0 openmpi/1.6.5 mallocmc/2.0.1 libSplash/1.2.4 adios/1.9.0 pngwriter/0.5.6 rivlib/1.0.0
```

Comment:
You removed the module `mallocmc/2.0.1` I think this is correct because we ship our own version of `mallocMC` in the directory `thirdPart` .